### PR TITLE
1.5 release page

### LINKED ISSUE #3132: Add release page for 1.5 to docs
Add a 1.5 page similar to this one:
https://www.kubeflow.org/docs/releases/kubeflow-1.4/

### PR DESCRIPTION
**Pull request by shannonbradshaw**
Fixes #3132

### CODE REVIEW

**Comment by shannonbradshaw** [comment]
@jbottum @annajung @kimwnasptd can I get reviews please?

File: content/en/docs/releases/kubeflow-1.5.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+## 1.5.0
+
+<div class="table-responsive">
+<table class="table table-bordered">
+  <tbody>
+    <tr>
+      <th class="table-light">Release Date</th>
+      <td>
+        2022-03-11
```
Comment (annajung) [inline content/en/docs/releases/kubeflow-1.5.md:15]: lgtm except the date of the 1.5 release
```suggestion
        2022-03-10
```

**Pull request merged**